src/main/java/filmorate/model/Film.java

```diff
@@ -28,4 +29,10 @@ public class Film {
 
     @Positive
     private int duration;
+
+    private Set<Long> usersLike;
+
+    public Film() {
+        this.usersLike = new HashSet<Long>();
```

Comment:
Компилятор уже знает, что мы будем создавать сет типа `Long`, второй раз прописывать уже необязательно
```suggestion
        this.usersLike = new HashSet<>();
```